### ISSUE TITLE
SA1503 fails to report `using` statement without braces

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#1185]
The following statement should result in SA1503, but currently does not produce any warning.

```csharp
using (this)
    Console.WriteLine();
```
